src/backend/renderer/gles/texture.rs

```diff
@@ -0,0 +1,140 @@
+use super::*;
+
+/// A handle to a GLES2 texture
+#[derive(Debug, Clone)]
+pub struct GlesTexture(pub(super) Rc<GlesTextureInternal>);
+
+impl GlesTexture {
+    /// Create a GlesTexture from a raw gl texture id.
+    ///
+    /// This expects the texture to be in RGBA format to be rendered
```

Comment:
Does this still apply? The function accepts a format so I would think it is possible to use a different format.